src/pjm/utils.jl

```diff
@@ -30,3 +30,100 @@ function get_str_dates(
     end_hour_minute = Dates.format(zd_end_date, "HH:MM")
     return start_date_str, start_hour_minute, end_date_str, end_hour_minute
 end
+
+"""
+    get_str_dates_file_name(start_date::ZonedDateTime, end_date::ZonedDateTime)::String
+
+Get the start and end date strings in the format "m-d-yyyy HH:MM" for the file name.
+
+# Arguments
+- start_date::ZonedDateTime: The start date of the data to be retrieved.
+- end_date::ZonedDateTime: The end date of the data to be retrieved.
+- market::PjmMarket: The market object.
+
+# Returns
+- String: The start and end date strings in the format "m-d-yyyy HH:MM" for the file name.
+"""
+function get_str_dates_file_name(
+    start_date::ZonedDateTime,
+    end_date::ZonedDateTime,
+    market::PjmMarket,
+)
+    zd_start_date = astimezone(start_date, market.timezone)
+    zd_end_date = astimezone(end_date, market.timezone)
+
+    start_date_str = Dates.format(zd_start_date, "m/d/yyyy HH:MM")
```

Comment:
start_date_str = Dates.format(zd_start_date, "m-d-yyyy HH:MM") works?

Reply:
No, this is the format of the url

Reply:
![image](https://github.com/user-attachments/assets/e6cbbb8a-1f96-410f-aca7-141aee8c578c)

Reply:
You’re right; that was a legacy from a previous version. I’ve just fixed it.